CONTRIBUTING.md

```diff
@@ -159,10 +158,6 @@ Once you've updated the test suite, there are a few things you'll want to check.
 You can also take a look at the `.travis.yml` file to see what the continuous
 integration system will do to verify the track.
 
-Take a look at our [pull request guidelines](#pull-request-guidelines).
-You don't need to get it perfect the first time around; we'll work with you to
-get the patch merged.
-
```

Comment:
But you have (had) extracted it from `CONTRIBUTING.md` and moved it into `PULL_REQUEST_TEMPLATE.md`  

I thought that they were still relevant in `CONTRIBUTING.md` to someone who is reading through that file who might not be making a pull request right now but might want to later.

I would probably suggest creating an explicit "Writing a Pull request" section in CONTRIBUTING.MD that contained the relevant information (this may still be a useful change.) and then referring to THAT from the Pull Request Template.

Another possibility would be to greatly simplify the `CONTRIBUTING.md` file which is linked to in the yellow box, and moving all the information that is not directly related to creating a pull request to a different file.